products/nginx.md

```diff
@@ -28,22 +28,31 @@ auto:
 # eol(x) = releaseDate(x+2)
 
 releases:
+-   releaseCycle: "1.27"
+    releaseDate: 2024-05-29
+    eol: false
+    link: https://nginx.org/en/CHANGES
+    latest: "1.27.0"
+    latestReleaseDate: 2024-05-29
+
 -   releaseCycle: "1.26"
     releaseDate: 2024-04-23
     eol: false
+    link: https://nginx.org/en/CHANGES
```

Comment:
The current link is better: https://nginx.org/en/CHANGES-1.26
```suggestion
```

Reply:
make sense 👍